src/phoenix/db/models.py

```diff
@@ -97,8 +97,8 @@ class Base(DeclarativeBase):
     # https://alembic.sqlalchemy.org/en/latest/naming.html#integration-of-naming-conventions-into-operations-autogenerate
     metadata = MetaData(
         naming_convention={
-            "ix": "ix_%(column_0_label)s",
-            "uq": "uq_%(table_name)s_%(column_0_name)s",
+            "ix": "ix_%(table_name)s_%(column_0_N_name)s",
+            "uq": "uq_%(table_name)s_%(column_0_N_name)s",
```

Comment:
- `column_0_label` refers to the table name and first column name joined by an underscore, e.g., `projects_name`
- `column_0_name` refers to the first column name, e.g., `name`
- `column_0_N_name` references to all columns joined by underscores, e.g., `span_rowid_document_position_name`

Reply:
[Here's](https://github.com/sqlalchemy/sqlalchemy/discussions/10464) someone hitting an issue with naming collisions for unique constraints, and the recommended resolution is to override the naming convention with manual names. I think it's simpler to just change the naming convention, though.